src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -290,7 +313,12 @@ const BorderlessEventEntries = styled(EventEntries)`
   }
 `;
 
-const StyledEventUserFeedback = styled(EventUserFeedback)`
+type StyledEventUserFeedbackProps = React.ComponentProps<typeof EventUserFeedback> & {
+  includeBorder: boolean;
+  theme?: any;
```

Comment:
as I mentioned above you don't need it, but for the next time you can use `Theme` -` import {Theme} from 'app/utils/theme';`

Reply:
Without declaring theme the ts does not build as I am accessing it on Line 321

Reply:
it already comes with the function styled :)